lib/MyUtils.groovy

```diff
@@ -0,0 +1,93 @@
+class MyUtils {
```

Comment:
Nice, very tidy! Maybe better to call `SarekUtils` instead of `MyUtils` though, so that it's clearer in `main.nf`?

Reply:
Good idea